ui/i18n/src/main/res/values/strings.xml

```diff
@@ -521,13 +521,16 @@
     <string name="pref_enqueue_downloaded_summary">Add downloaded episodes to the queue</string>
     <string name="pref_skip_silence_title">Skip silence in audio</string>
     <string name="behavior">Behavior</string>
+    <string name="episode_lists">Episode Lists</string>
     <string name="pref_default_page">Default page</string>
     <string name="pref_default_page_sum">Screen that is opened when starting AntennaPod</string>
     <string name="pref_back_button_opens_drawer">Back button opens drawer</string>
     <string name="pref_back_button_opens_drawer_summary">Pressing the back button on the default page opens the navigation drawer</string>
     <string name="remember_last_page">Remember last page</string>
     <string name="pref_delete_removes_from_queue_title">Delete removes from queue</string>
     <string name="pref_delete_removes_from_queue_sum">Automatically remove an episode from the queue when it is deleted</string>
+    <string name="pref_downloads_button_action_title">Switch Delete button to Play/Pause</string>
```

Comment:
I would turn this setting around. The default across the app is the play button. So it should say something like "show delete button on downloads page" and be enabled by default. Calling it something along the lines of "switch" would assume that users know how it was before.

Reply:
Hmm, not sure.
Generally, I think that settings should be off by default. Activating a setting enables a different use-case.
With that in mind, the default for this specific screen shouldn't necessarily be the same as the rest of the app. The purpose of the screen is different, after all, and I'd keep it that way: the default purpose being 'manage the episodes' rather than take over the purpose of the queue. If users first open the app seeing play buttons they're more likely to start using it as the queue, then complain that continuous playback is not working and request that we introduce more sorting options.

Reply:
> I think that settings should be off by default

I would rather have a setting that enables something than a setting that is basically a double negation ("Enable to not have the thing" / "disable to not-not have the thing"). On Android, many settings are enabled by default. For example, it is "Automatic screen rotation", not "disable rotation" or " Lock rotation".

Something like "Do X" is easier to comprehend than "Replace Y with X", no matter what the expected default is.

Reply:
> I would rather have a setting that enables something than a setting that is basically a double negation ("Enable to not have the thing" / "disable to not-not have the thing")

Agreed. I believe I was not suggesting a double negation? In fact, I believe I was propose a setting that _enables_ something.

> Something like "Do X" is easier to comprehend than "Replace Y with X", no matter what the expected default is.

Sure, but that's if it's an on or off setting. I don't think that's really the case here; we're not switching the 'delete' (as per your proposal) or 'play' button (as per my proposal) on or off. Whatever default we pick, we switch between A and B.

For your proposal you assume that users immediately/quickly deduct that the 'Play' button is the app's default. I think that's wrong to assume, though: Firstly, for new users, because the default for this screen (what the user will see when they use the screen for the first time) is the Delete button. Secondly, even if we assume the user never saw this specific screen before, I think you're overestimating users if you believe that they will think 'the other screens have play buttons so the Downloads screen will have one too' (rather, I expect them not to do such deduction, even unconsciously). Thirdly, for existing users, because they have been used to the Downloads screen having the Delete button.

Either way, my proposal "Play/Pause button on Downloads screen" is very close to your "Do X", if you ask me.

Let's compare these two:
* "Play/Pause button on Downloads screen"
  * Off by default
  * User knows (from seeing the screen once) that the default state on that screen = the Delete button
  * User knows from the setting title that they  can _enable_ the Play button instead
  * The setting description serves as reminder of the original state
* "Delete button on Downloads screen"
  * On by default
  * User knows both from seeing the screen once and from the setting title that the default state on that screen = the Delete button
  * User has no way to know how they can enable the play button instead, except by reading the setting description (as noted above we cannot expect the user to deduct from the rest of the app that the 'default' is the Play button (which by default is overwritten for this screen))
  * User knows from the setting title how they can _revert the enabling_ of the Delete button (which is not quite "Do X")